### PR TITLE
AGWG Updates

### PR DESCRIPTION
**Pull request by WilcoFiers**
Resolve feedback from this old AGWG survey:
- https://www.w3.org/2002/09/wbs/35422/act-dec-2022/results

Need for Call for Review: none, editorial

---

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Call for Review period. In case of disagreement, the longer period wins.

### CODE REVIEW

**Review by Jym77 — APPROVED** [review]

File: _rules/html-page-title-descriptive-c4a8a4.md [hunk 1]
```diff
@@ -37,7 +37,10 @@ htmlHintIgnore:
 
 ## Applicability
 
-This rule applies to the [document title][] of each [html web page][] if this [document title][] contains not only [whitespace](#whitespace) [text nodes](https://dom.spec.whatwg.org/#text).
+This rule applies to the [document title][] of each [html web page][], except if one of the following is true:
+
+- The [html web page][] has no [document title][]; or
+- The [document title][] contains not only [whitespace](#whitespace) [text nodes](https://dom.spec.whatwg.org/#text).
```
Comment (tbostic32) [inline _rules/html-page-title-descriptive-c4a8a4.md]:
```suggestion
- The [document title][] contains only [whitespace](#whitespace) [text nodes](https://dom.spec.whatwg.org/#text).
```

File: _rules/image-button-non-empty-accessible-name-59796f.md [hunk 2]
```diff
@@ -60,7 +60,7 @@ The [input type="image" Accessible Name Computation algorithm](https://www.w3.or
 
 ## Background
 
-Contrarily to `img` elements, an empty `alt` attribute does not make image button decorative; image buttons have a button role and are therefore exposed as interactive elements. Consequently, an empty `alt` attribute does not provide a "usable string" for image buttons and the computation defaults to other means of providing a name, as defined in [input type="image" Accessible Name Computation algorithm](https://www.w3.org/TR/html-aam/#input-type-image-accessible-name-computation).
+Contrarily to `img` elements, an empty `alt` attribute (`alt=""`) does not make image button decorative; image buttons have a button role and are therefore exposed as interactive elements. Consequently, an empty `alt` attribute does not provide a "usable string" for image buttons and the computation defaults to other means of providing a name, as defined in [input type="image" Accessible Name Computation algorithm](https://www.w3.org/TR/html-aam/#input-type-image-accessible-name-computation).
```
Comment (tbostic32) [inline _rules/image-button-non-empty-accessible-name-59796f.md]:
```suggestion
Contrarily to `img` elements, an empty `alt` attribute (`alt=""`) does not make an image button decorative; image buttons have a button role and are therefore exposed as interactive elements. Consequently, an empty `alt` attribute does not provide a "usable string" for image buttons and the computation defaults to other means of providing a name, as defined in [input type="image" Accessible Name Computation algorithm](https://www.w3.org/TR/html-aam/#input-type-image-accessible-name-computation).
```

File: _rules/html-page-title-descriptive-c4a8a4.md (hunk 1 above)
Comment (kengdoj) [reply _rules/html-page-title-descriptive-c4a8a4.md]: +1 to trevor's suggestion

File: _rules/image-non-empty-accessible-name-23a2a8.md [hunk 3]
```diff
@@ -52,7 +52,7 @@ There are no assumptions.
 
 ## Accessibility Support
 
-- There are several popular browsers that do not treat images with empty `alt` attribute as having a role of `presentation` but instead add the `img` element to the accessibility tree with a [semantic role][] of either `img` or `graphic`.
+- There are several popular browsers that do not treat images with empty `alt` attribute (`alt=""`) as having a role of `presentation` but instead add the `img` element to the accessibility tree with a [semantic role][] of either `img` or `graphic`.
```
Comment (tbostic32) [inline _rules/image-non-empty-accessible-name-23a2a8.md]:
```suggestion
- There are several popular browsers that do not treat images with an empty `alt` attribute (`alt=""`) as having a role of `presentation` but instead add the `img` element to the accessibility tree with a [semantic role][] of either `img` or `graphic`.
```

**Review by kengdoj — DISMISSED** [review]
just Trevor's suggestion

File: _rules/html-page-title-descriptive-c4a8a4.md (hunk 1 above)
Comment (daniel-montalvo) [reply _rules/html-page-title-descriptive-c4a8a4.md]: +1

File: _rules/image-button-non-empty-accessible-name-59796f.md (hunk 2 above)
Comment (daniel-montalvo) [reply _rules/image-button-non-empty-accessible-name-59796f.md]: Yes, either "an image button" or "images buttons". I'd prefer the latter for consistency with the next occurrence.

**Review by daniel-montalvo — DISMISSED** [review]

**Review by daniel-montalvo — APPROVED** [review]

**Review by tbostic32 — APPROVED** [review]

**Pull request merged**